src/content/docs/logs/ui-data/timestamp-support.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Understanding and using timestamps in logs
+tags:
+  - Logs
+  - Timestamp
+  - Timestamps
+  - Forwarding Logs
+metaDescription: "How New Relic handles timestamps in logs."
+---
+
+## Introduction [#introduction]
+
+One of the most important attributes in a log event is the **timestamp**.  It is the exact moment in time that tells when the event happened and is critical for ordering events in chronological order as well as for searching logs in specific timeframes.  The most basic question all logs answer is **what happened, and when** it happened.  In most cases, the timestamp will appear somewhere in the beginning of the log event, but this is not always the case.  In some cases, it may appear much later in the log.  In logs that are in a JSON format they will usually contain some attribute that identifies the timestamp, like `timestamp` or `log_timestamp`, `time`, etc.  Some logs may not contain timestamps at all, and in these cases New Relic will always assign a timestamp automatically to the event at the time it is ingested into the system.  New Relic also indexes the timestamp of each log, since they are crucial for efficient querying.
```

Comment:
For this sentence: Some logs may not contain timestamps at all, and in these cases New Relic will always assign a timestamp automatically to the event at the time it is ingested into the system.

Isn't this the default behavior anyway? Do we need to call this out if New Relic is always going to add the ingest time of the timestamp unless there's an override set up?

Reply:
It is the default behavior, I'm fine with removing this if you don't think it's necessary.  Was just trying to point out the default behavior somewhere in the intro even though I do call it out in the very beginning of the next section.